src/gm3/actions/mapSource.js

```diff
@@ -658,9 +657,9 @@ export function changeFeatures(mapSourceName, filter, properties) {
  *
  */
 export function getOrderedMapSources(mapSources) {
-    let active = [];
-    for(let ms in mapSources) {
-        if(isMapSourceActive(map_sources[ms])) {
+    const active = [];
```

Comment:
How did this work before?

Reply:
I'm not sure it did! Or it worked due to a scope issue with `var`